_overviews/scala3-scaladoc/static-site.md

```diff
@@ -37,6 +37,15 @@ getting-started.html
 
 Scaladoc can transform both files and directories (to organize your documentation into a tree-like structure). By default, directories have a title based on the file name and have empty content. It is possible to provide index pages for each section by creating `index.html` or `index.md` (not both) in the dedicated directory.
 
+Before generating your static site you also need to set the `-siteroot` to the name of your `<site-root>`,which means that when you generate documentation for your project, the root URL for the generated documentation will be `<site-root>`.
+The configuration is as follows:
```

Comment:
```suggestion
Before generating your static site you need to set the `-siteroot` value in your doc `scalacOptions`. The value of this is the directory that holds your docs.  The root URL for the generated documentation will also be `<site-root>`.
The configuration is as follows:
```

Reply:
Do you think it's okay if I detail it like this? Or should I just keep the sentence as you did.
For example if you have a directory called `docs` and you'd like that to be treated as your site root :

```
.
└── docs/
    └── _docs/
        ├── index.html
        └── getting-started.html
```

Then the configuration would be as follows: